src/m365/planner/commands/task/task-get.ts

```diff
@@ -24,30 +32,201 @@ class PlannerTaskGetCommand extends GraphCommand {
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    this
+      .getTaskId(args.options)
+      .then(taskId => {
+        const requestOptions: any = {
+          url: `${this.resource}/beta/planner/tasks/${encodeURIComponent(taskId)}`,
```

Comment:
Do we need the beta endpoint here for this call?

Reply:
This one uses the beta endpoint because of this issue #3037